deposit/bytecode-verification/deposit-spec.ini

```diff
@@ -55,55 +55,55 @@ VYPER_GENERATED_BOUNDS:
+    [ 536 /* = 320 + 32 *  6 + 24 */ := #buf(32, 8) ]              /* for slice(convert(y, bytes32), start=24, len=8) */
+    [ 704 /* = 320 + 32 * 12      */ := #buf(32, 8) ++ #bufSeg(#buf(32, Y8), 24, 8) ]  /* return: bytes[8] */ /* memcpy(704 <- 536, 40) */
     {TO_LITTLE_ENDIAN_64_ZERO_PADDING}
 LOCAL_MEM_UPDATE_BY_TO_LITTLE_ENDIAN_64_DEPOSIT_AMOUNT:
-    [ 352 := #buf(32, {RETURN_ADDR_FROM_TO_LITTLE_ENDIAN_64_DEPOSIT_AMOUNT}) ]  /* return address */
-    [ 320 := #buf(32, {DEPOSIT_AMOUNT}) ]   /* argument */
-    [ 384 := #buf(32, YY8) ]                /* y: uint256 = (y << 8) + bitwise_and(x, 255) */
-    [ 416 := #buf(32, XX8) ]                /* x: uint256 = (x >> 8) */
-    [ 448 := #buf(32, 8) ]                  /* for-loop index */
-    [ 480 := #buf(32, XX7 &Int 255) ]       /* bitwise_and(x, 255) */
-    [ 544 := #buf(32, YY8) ]                /* memcpy(544 <- 384, 32) */
-    [ 536 := #buf(32, 8) ]                  /* for slice(convert(y, bytes32), start=24, len=8) */
-    [ 704 := #buf(32, 8) ++ #bufSeg(#buf(32, YY8), 24, 8) ]  /* return: bytes[8] */ /* memcpy(704 <- 536, 40) */
+    [ 352 /* = 320 + 32 *  1      */ := #buf(32, {RETURN_ADDR_FROM_TO_LITTLE_ENDIAN_64_DEPOSIT_AMOUNT}) ]  /* return address */
+    [ 320 /* = 320 + 32 *  0      */ := #buf(32, {DEPOSIT_AMOUNT}) ]   /* argument */
+    [ 384 /* = 320 + 32 *  2      */ := #buf(32, YY8) ]                /* y: uint256 = (y << 8) + bitwise_and(x, 255) */
+    [ 416 /* = 320 + 32 *  3      */ := #buf(32, XX8) ]                /* x: uint256 = (x >> 8) */
+    [ 448 /* = 320 + 32 *  4      */ := #buf(32, 8) ]                  /* for-loop index */
+    [ 480 /* = 320 + 32 *  5      */ := #buf(32, XX7 &Int 255) ]       /* bitwise_and(x, 255) */
+    [ 544 /* = 320 + 32 *  7      */ := #buf(32, YY8) ]                /* memcpy(544 <- 384, 32) */
+    [ 536 /* = 320 + 32 *  6 + 24 */ := #buf(32, 8) ]                  /* for slice(convert(y, bytes32), start=24, len=8) */
+    [ 704 /* = 320 + 32 * 12      */ := #buf(32, 8) ++ #bufSeg(#buf(32, YY8), 24, 8) ]  /* return: bytes[8] */ /* memcpy(704 <- 536, 40) */
     {TO_LITTLE_ENDIAN_64_ZERO_PADDING}
 TO_LITTLE_ENDIAN_64_ZERO_PADDING:
-    [ 800 := #buf(32, 32) ]             /* padding loop index */
-    [ 744 <- 0 ]                        /* padding */
-    [ 745 <- 0 ]
-    [ 746 <- 0 ]
-    [ 747 <- 0 ]
-    [ 748 <- 0 ]
-    [ 749 <- 0 ]
-    [ 750 <- 0 ]
-    [ 751 <- 0 ]
-    [ 752 <- 0 ]
-    [ 753 <- 0 ]
-    [ 754 <- 0 ]
-    [ 755 <- 0 ]
-    [ 756 <- 0 ]
-    [ 757 <- 0 ]
-    [ 758 <- 0 ]
-    [ 759 <- 0 ]
-    [ 760 <- 0 ]
-    [ 761 <- 0 ]
-    [ 762 <- 0 ]
-    [ 763 <- 0 ]
-    [ 764 <- 0 ]
-    [ 765 <- 0 ]
-    [ 766 <- 0 ]
-    [ 767 <- 0 ]
-    [ 672 := #buf(32, 32) ]             /* ??? */
-    [ 640 := #buf(32, 0) ]              /* ??? */
+    [ 800 /* = 320 + 32 * 15      */ := #buf(32, 32) ]             /* padding loop index */
+    [ 744 /* = 320 + 32 * 13 +  8 */ <- 0 ]                        /* padding */
```

Comment:
`[ 744 /* = 320 + 32 * 13 +  8 */ := #buf(22, 0) ]` would rewrite to this, no? That seems cleaner to me

Reply:
Yes, they are equivalent, but the current form is easier for K to prove.